REUSE.toml

```diff
@@ -0,0 +1,16 @@
+version = 1
+SPDX-PackageName = "Gardener gardener-extension-registry-cache"
+SPDX-PackageSupplier = "The Gardener project <gardener@googlegroups.com>"
+SPDX-PackageDownloadLocation = "https://github.com/gardener/gardener-extension-registry-cache"
+
+[[annotations]]
+path = [".dockerignore", ".github/**", ".gitignore", ".golangci.yaml", ".test-defs/**", "OWNERS", "OWNERS_ALIASES", "VERSION", "charts/**", "example/**", "go.mod", "go.sum", "hack/api-reference/**.json", "local-setup/**", "pkg/component/registrycaches/monitoring/dashboard.json", "pkg/component/registrycaches/templates/config.yml.tpl", "skaffold-operator.yaml", "skaffold.yaml"]
```

Comment:
Should we also put every entry in a new line for a better readability and easier maintenance?

/hold